### PR TITLE
AB#65223 Accept he terms and conditions for account

### PR DESCRIPTION
**Pull request by dawid-pawulski**

### CODE REVIEW

File: src/HE.InvestmentLoans.BusinessLogic/User/CommandHandlers/ProvideAcceptHeTermsAndConditionsCommandHandler.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    public ProvideAcceptHeTermsAndConditionsCommandHandler(ILogger<ProvideUserDetailsCommandHandler> logger)
+    {
+        _logger = logger;
+    }
+
+    public async Task<OperationResult> Handle(ProvideAcceptHeTermsAndConditionsCommand request, CancellationToken cancellationToken)
+    {
+        try
+        {
+            await Task.Run(() => AcceptHeTermsAndConditions.FromString(request.HeTermsAndConditions), cancellationToken);
```
Comment (Carq) [inline src/HE.InvestmentLoans.BusinessLogic/User/CommandHandlers/ProvideAcceptHeTermsAndConditionsCommandHandler.cs:23]: Remove Task.Run. Just Add `Task.FromResult()` with return statments.

File: src/HE.InvestmentLoans.WWW/Views/Home/AcceptHeTermsAndConditions.cshtml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+@using HE.InvestmentLoans.Common.Utils.Enums
+@model AcceptHeTermsAndConditionsModel;
+
+<gds-div-grid-column-two-thirds>
```
Comment (Carq) [inline src/HE.InvestmentLoans.WWW/Views/Home/AcceptHeTermsAndConditions.cshtml:4]: Did you checked WCAG?

**Review by Carq — APPROVED** [review]

File: src/HE.InvestmentLoans.BusinessLogic/User/CommandHandlers/ProvideAcceptHeTermsAndConditionsCommandHandler.cs (hunk 1 above)
Comment (dawid-pawulski) [reply src/HE.InvestmentLoans.BusinessLogic/User/CommandHandlers/ProvideAcceptHeTermsAndConditionsCommandHandler.cs]: Done

File: src/HE.InvestmentLoans.WWW/Views/Home/AcceptHeTermsAndConditions.cshtml (hunk 2 above)
Comment (dawid-pawulski) [reply src/HE.InvestmentLoans.WWW/Views/Home/AcceptHeTermsAndConditions.cshtml:7]: Yep

**Review by PiotrSamb — APPROVED** [review]

**Pull request merged**